CONTRIBUTING.md

```diff
@@ -1,6 +1,28 @@
+Before reporting a bug
+---
+1. Search [issue tracker](https://github.com/aframevr/aframe/issues) for similar issues. 
+2. Try the latest dev branch version of [aframe](https://github.com/aframevr/aframe/tree/dev) and [aframe-core](https://github.com/aframevr/aframe-core/tree/dev).
+
+How to report a bug
+---
+1. Specify the revision number of the aframe.js library where the bug occurred.
+2. If you are using a PC, specify your browser version, operating system, and graphics card (for example, Chrome 23.0.1271.95, Windows 7, Nvidia Quadro 2000M).
+3. If you are using a mobile device, specify your make, model, operating system version, and browser (for example, Samsung Galaxy Nexus, Android 4.4.4, Chrome 47.0.2526.83).
+3. Describe the problem in detail. Explain what happened, and what you expected would happen.
+4. Provide a small test-case (http://jsfiddle.net). [Here is a fiddle](http://jsfiddle.net/8tvqo8jh/) that uses the dev branch. If a test-case is not possible, provide a link to a live version of your application.
```

Comment:
the only concern I have with recommending JS Fiddle is that you can't enter VR from the Edit mode (only in presentation mode). see https://github.com/aframevr/aframe-core/issues/725